spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -787,19 +780,43 @@ contract PerpetualTranche is
         uint256 trancheInAmtAvailable,
         uint256 tokenOutAmtRequested
     ) private returns (IPerpetualTranche.RolloverData memory) {
+        //-----------------------------------------------------------------------------
+        // Rollover fees have to be settled with all callers.
```

Comment:
Sure we can get rid of this. I was just hoping to convey, that unlike mint and burn fees which can be short-circuited when called from the vault, rollover fees cannot. However since the vault is the only caller, it doesn't really make sense.